src/mobilenet.jl

```diff
@@ -0,0 +1,296 @@
+                       conv_bn((1, 1), channels // reduction, channels, hardσ)...,), .*)
+
+"""
+    invertedresidualv3(inplanes, hidden_planes, outplanes, kernel_size, stride, use_se, use_hs)
+
+Create a basic inverted residual block for MobileNetv3
+([reference](https://arxiv.org/abs/1905.02244)).
+
+# Arguments
+- `inplanes`: The number of input feature maps
+- `hidden_planes`: The number of feature maps in the hidden layer
+- `outplanes`: The number of output feature maps
+- `kernel_size`: The kernel size of the convolutional layers
+- `stride`: The stride of the convolutional kernel, has to be either 1 or 2
+- `use_se`: If `true`, Squeeze and Excitation layer will be used
+- `use_hs`: If `true`, Hard-Swish activation function will be used
+"""
+function invertedresidualv3(inplanes, hidden_planes, outplanes, kernel_size,
+                            stride, use_se, use_hs)
+  @assert stride in [1, 2] "`stride` has to be 1 or 2"
+
+  if inplanes == hidden_planes
+    invres = Chain(conv_bn((kernel_size, kernel_size), hidden_planes, hidden_planes, use_hs ? hardswish : relu;
+                            bias = false, stride, pad = (kernel_size - 1) ÷ 2, groups = hidden_planes)...,
+                            use_se ? selayer(hidden_planes) : identity,
+                   conv_bn((1, 1), hidden_planes, outplanes, identity; bias = false)...)
+  else
+    invres = Chain(conv_bn((1, 1), inplanes, hidden_planes, use_hs ? hardswish : relu; bias = false)...,
+                   conv_bn((kernel_size, kernel_size), hidden_planes, hidden_planes, use_hs ? hardswish : relu;
+                            bias = false, stride, pad = (kernel_size - 1) ÷ 2, groups = hidden_planes)...,
+                            use_se ? selayer(hidden_planes) : identity,
+                   conv_bn((1, 1), hidden_planes, outplanes, identity; bias = false)...)
+  end
+
+  (stride == 1 && inplanes == outplanes) ? SkipConnection(invres, +) : invres
+end
+
+"""
+    mobilenetv3(configs, width_mult; max_width = 1024, nclasses = 1000)
+
+Create a MobileNetv3 model.
+([reference](https://arxiv.org/abs/1905.02244)).
+
+# Arguments
+- `configs`: a "list of tuples" configuration for each layer that details:
+  - `k::Int` - The size of the convolutional kernel
+  - `c::Float` - The multiplier factor for deciding the number of feature maps in the hidden layer
+  - `t::Int` - The number of output feature maps for a given block
+  - `use_se::Bool` - Whether to use Squeeze and Excitation layer
+  - `use_hs::Bool` - Whether to use Hard-Swish activation function
+  - `s::Int` - The stride of the convolutional kernel
+- `width_mult`: Controls the number of feature maps in each layer, with 1.0 being the original
+  model as detailed in the paper.
+- `max_width`: The maximum number of feature maps in any layer of the network
+- `nclasses`: the number of output classes
+"""
+function mobilenetv3(configs, width_mult; max_width = 1024, nclasses = 1000)
```

Comment:
I see the reasoning for `config` first in this case, but I think consistency with v2 outweighs it.
```suggestion
function mobilenetv3(width_mult, configs; max_width = 1024, nclasses = 1000)
```